src/content/docs/en/reference/dev-toolbar-app-reference.mdx

```diff
@@ -434,6 +434,12 @@ The style of the button (`ghost`, `outline`, `purple`, `gray`, `red`, `green`, `
 
 In JavaScript, set this property using the `buttonStyle` property to avoid conflict with the native `style` property.
 
+#### `button-border-radius`
+
+The border radius of the button (`normal`, `rounded`). When using `rounded`, the button will have rounded corners and homogenous padding on all sides.
```

Comment:
```suggestion
The border radius of the button (`normal`, `rounded`). When using `rounded`, the button will have rounded corners and uniform padding on all sides.
```
I think "uniform" or even "equal" padding feels a bit more natural. Any other word you might use instead?

Reply:
uniform is fine with me!